brion/plugin/compartmentReportBinary.cpp

```diff
@@ -706,23 +716,23 @@ bool CompartmentReportBinary::_parseMapping()
                 const uint64_t frameIndex =
                     j + ((info.dataOffset - _header.dataBlockOffset) /
                          sizeof(float));
```

Comment:
You do realize, of course, that using sizeof(float) for dealing with locally allocated memory and for something created externally is a different story.
If the spec says it's 4 bytes for measurement in the report I would rather see the constant '4'.
But I'm not an expert on machine-dependent float type size to insist on that.